functests-performance-profile-creator/1_performance-profile_creator/ppc.go

```diff
@@ -174,3 +271,17 @@ func getExpectedProfiles(expectedProfilesPath string, mustGatherDirs map[string]
 
 	return expectedProfiles
 }
+
+// PPC Script stderr parser
+func errorStringParser(err error) string {
```

Comment:
I'd rather fix `functests/utils/utils.go:ExecAndLogCommand`, but not critical

Reply:
Like: https://github.com/openshift-kni/performance-addon-operators/pull/658 - fully untested

Reply:
I have not fixed this in the latest PR. I can fix it in the next iteration.

Reply:
works for me. Can be done later.